### ISSUE TITLE
Unable to send activation email

### ISSUE THREAD

**Issue by tsaoyu** [WRSC/tracking#23]
The tracking website is not able to deliver activation email to the teams. 

![image](https://user-images.githubusercontent.com/6488896/44309457-d9eb7280-a3be-11e8-99cc-adce94784d96.png)

**Comment by tsaoyu** [comment]
This might due to use of the development environment in the production. 
However, there is still an internal server error when I switch to production mode. 

So for this year competition, I will create an account for each team using seed file beforehand.

**Comment by zysim** [comment]
**TL;DR** [Go here](https://www.google.com/settings/security/lesssecureapps) and check the box.

The 500 error is due to Gmail's security settings. [This guide](https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration-for-gmail) shows how to solve this issue; specifically [this link](https://www.google.com/settings/security/lesssecureapps), which, once set, will allow "less secure apps" to access the email. This of course means we need access to the Gmail account in the first place.

I've tested a production build on my local machine, using my own email, and configuring that Gmail setting, and it works; the email is successfully sent.

However, it seems that the page doesn't redirect to the "please check your email for the activation link" page. `production.log` spits out a `FileNotFound` error too. I'll look into this tomorrow.

**Comment by zysim** [comment]
Okay so my previous `FileNotFound` error is gone. Everything works as usual. Oh well.
